python/hsfs/core/statistics_engine.py

```diff
@@ -176,7 +176,7 @@ def get(
         training_dataset_version=None,
     ):
         """Get Statistics with the specified commit time of an entity."""
```

Comment:
FeatureGroup also has a function `def get_statistics(self, commit_time: str = None):` which should be updated in a similar way.

Reply:
this is converted in statistics engine `get` mothod

Reply:
Okay, but then the type hints need to be updated

Reply:
done both for FG and TD

Reply:
Looks like this was done